c2corg_api/scripts/es/syncer.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import sys
+import os
+from c2corg_api.scripts.es.sync import sync_es
+from c2corg_api.search import configure_es_from_config, get_queue_config
+from pyramid.scripts.common import parse_vars
+from pyramid.paster import get_appsettings, setup_logging
+from sqlalchemy import engine_from_config
+
+from kombu.mixins import ConsumerMixin
+from sqlalchemy.orm import sessionmaker
+
+log = logging.getLogger('c2corg_api_syncer')
+
+
+class SyncWorker(ConsumerMixin):
+    """Worker that listens to messages in the Redis queue and then synchronizes
+    changes that have been made in the Postgres database with ElasticSearch.
+
+    Based on this example:
+    http://docs.celeryproject.org/projects/kombu/en/latest/userguide/examples.html
+    """
+
+    def __init__(self, connection, queue, session=None, session_factory=None):
+        self.connection = connection
+        self.queue = queue
+        self.session = session
+        self.session_factory = session_factory
+
+    def get_consumers(self, consumer_factory, channel):
+        return [consumer_factory(
+            queues=[self.queue], callbacks=[self.process_task])]
+
+    def process_task(self, body, message):
+        log.info('Sync requested')
+        message.ack()
```

Comment:
Do you want failed messages to be removed from the queue (no retry)?
If yes, then I think you should add a comment saying so.